.docs/content/1.installation/3.aws/2.aws-deployment-troubleshooting.md

```diff
@@ -0,0 +1,115 @@
+# AWS Deployment Troubleshooting Guide 🛠️
+
+This guide covers common issues you might encounter while deploying your application on AWS and provides solutions to address them.
+
+## 1. Error: Partition Not Found
+
+If you encounter a Partition Not Found error during deployment or runtime, ensure the following:
+
+- Set the Replication Factor to 1 in the [`parameters.tfvars`](https://github.com/aneoconsulting/ArmoniK/blob/main/infrastructure/quick-deploy/aws/parameters.tfvars):
+
+```hcl
+default = {
+    node_selector = { service = "workers" }
+    replicas = 1 # HERE
+}
+```
+
+- Check Spelling in Configuration Files.
+- Double-check for any typos in partition names or configurations. Even minor spelling mistakes can cause the error.
+
+For example in [`parameters.tfvars`](https://github.com/aneoconsulting/ArmoniK/blob/main/infrastructure/quick-deploy/aws/parameters.tfvars):
+
+```hcl
+default = {
+    node_selector = { service = "workers" }   
+    replicas = 1 # HERE
+```
+
+instead of
+
+```hcl
+defautl = {
+    node_selector = { service = "workers" }   
+    replicas = 1 # HERE
+
+```
+
+## 2. Error: Incorrect URL or Missing Endpoint
+
+If your application fails to connect to a service, verify the following:
+
+### Use the Correct URL Generated by AWS
+
+After deploying your application or service, AWS typically generates a URL for the *control_plane* or other services. Make sure to:
+
+- Copy the URL provided by AWS during deployment.
+- Update the client configuration with the correct endpoint before building your project.
```

Comment:
not sure to understand this sentence. The client is not started as an executable with command line option? The ip is written on hard and so need the client to be rebuild?